src/app/admin/components/PopupTileUser.tsx

```diff
@@ -0,0 +1,170 @@
+"use client";
+
+import { generateClient } from "aws-amplify/data";
+import Image from "next/image";
+import { Bounce, toast } from "react-toastify";
+
+import { type Schema } from "@/amplify/data/resource";
+import { useMutation } from "@tanstack/react-query";
+
+const exit_icon = "/svgs/admin/exit_icon.svg";
+
+const VIEW_TEAM_POPUP_SECTION_STYLES =
+  "fixed left-0 top-0 flex size-full items-center justify-center bg-black/60";
+const VIEW_TEAM_POPUP_TILE_STYLES =
+  "w-4/5 max-w-[1200px] rounded-md bg-white p-6";
+
+const VIEW_TEAM_POPUP_HEADER_STYLES = "mb-4 flex justify-between";
+
+const VIEW_TEAM_POPUP_TABLE_SECTION_STYLES =
+  "rounded-md borderborder-awesomer-purple bg-light-grey p-2";
+const VIEW_TEAM_POPUP_TABLE_CONTENT_STYLES =
+  "w-full border-separate border-spacing-2 text-left";
+const VIEW_TEAM_POPUP_TABLE_HEADER_STYLES = "bg-awesome-purple text-white";
+const VIEW_TEAM_POPUP_TABLE_CELL_STYLES = "rounded-md p-2";
+
+const DELETE_RECORD_POPUP_TILE_STYLES = "w-2/5 bg-white p-8 rounded-md ";
+
+interface PopupProps {
+  selectedMembersData: string[];
+  selectedMemberStatus: string | string[];
+  popupType: string;
+  teamName: string;
+  recordToDelete: string | undefined;
+  onClose: () => void;
+}
+
+const client = generateClient<Schema>();
+
+const PopupUser = ({
```

Comment:
I think we can do a better job with deleting a user? Instead of passing in just the users id, we could pass in the whole user, query dynamo for the members of the team and the team name, and show it that way?

Reply:
Yeah I agree, this is a little confusing, especially since we don't care about Team Id when deleting just a user. Is this because this is being shared between user and team?

Reply:
Well there is the edge case where the user that is being deleted is the last one in a team. If the team is empty, I think we should just delete the team entirely

Reply:
I think we can make another ticket for this once we start testing the app @ideen1